config/redis.yml

```diff
@@ -1,9 +1,9 @@
 development:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
+  port: <%= ENV['REDIS_PORT'] || 6379 %>
 test:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
+  port: <%= ENV['REDIS_PORT'] || 6379 %>
 production:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
```

Comment:
This is documented here for a future work: https://github.com/ualbertalib/Hydranorth2/issues/142